x/icacallbacks/migrations/v2/convert.go

```diff
@@ -74,14 +74,14 @@ func convertCallbackData(oldCallbackData icacallbacktypes.CallbackData) (icacall
 		// Deserialize the callback args with the old DelegateCallback type
 		oldDelegateCallback := oldstakeibctypes.DelegateCallback{}
 		if err := proto.Unmarshal(oldCallbackData.CallbackArgs, &oldDelegateCallback); err != nil {
-			return icacallbacktypes.CallbackData{}, errorsmod.Wrapf(stakeibctypes.ErrUnmarshalFailure, err.Error())
+			return icacallbacktypes.CallbackData{}, errorsmod.Wrapf(err, "failed to unmarshal")
```

Comment:
Nit: this and all the below should be Wrap

Reply:
done